packages/compass-connections/src/connections-manager.ts

```diff
@@ -307,6 +307,14 @@ export class ConnectionsManager extends EventEmitter {
     }
   }
 
+  async closeAllConnections() {
```

Comment:
Is this something we could use to clean-up when we quit compass or close a browser window? 

cc: @himanshusinghs @paula-stacho

Reply:
I'm working on COMPASS-7894 and can add a call to this in compass-connections unmount, but also if I recall correctly, window process closing will force close all the outgoing connections, similar to stopping a Node.js process that opened a driver connection to somewhere